src/frontend/src/lib/components/networks/NetworkButton.svelte

```diff
@@ -27,8 +29,13 @@
 	};
 </script>
 
-<button class="w-full flex justify-between" on:click={onClick}>
-	<TextWithLogo {name} {description} {icon} logo="start" />
+<button class="w-full flex justify-between items-start" on:click={onClick}>
```

Comment:
@peterpeterparker in one of the previous PRs, we removed `items-start` from this line, but it appears there is a global `button.scss` file which adds `align-items: center` to all `button`s. In this particular case, I need to have `align-items: flex-start`, so I added back that class.